test/e2e/framework/cluster.go

```diff
@@ -108,3 +111,17 @@ func runCmd(command string, args ...string) error {
 
 	return nil
 }
+
+// GetKnodeClient get knode client
+func GetKnodeClient(client versioned.Interface, clusterName string) kubernetes.Interface {
```

Comment:
parameter `clusterName` make me confused,  `knodeName` maybe  more suitable

Reply:
ok，i will fix it